src/consensus/blockstat.go

```diff
@@ -571,11 +565,6 @@ func (self *BlockStatQueue) try_append_to_BlockStatQueue(
 	}
 	n = -1 // guard
 
-	if !(action_update || action_skip || action_insert) {
```

Comment:
Comment out this code instead (normally you should delete it, but this code in the `consensus/` folder is reference code and I do not want to remove what was there)